circle.yml

```diff
@@ -94,9 +142,16 @@ workflows:
       - e2e:
           requires: [build]
 
-      - release:
-          context: npm-library
+      - approve-release:
+          type: approval
           requires: [build, lint, unit, e2e]
           filters:
             branches:
               only: master
+
+      - release:
+          context: npm-library
+          requires: [approve-release]
```

Comment:
Perhaps instead of requiring `approve-release`, you can remove the approve-release step and add `requires` and `filters` to this step.

Reply:
Update: it turns out this is necessary.